.github/workflows/ci.yml

```diff
@@ -326,6 +326,76 @@ jobs:
                 ./build_tools/scripts/check_vulkan.sh
                 ./build_tools/cmake/ctest_all.sh ${BUILD_DIR}"
 
+  test_nvidia_a100_presubmit:
+    env:
+      IREE_CPU_DISABLE: 1
+      IREE_VULKAN_DISABLE: 0
+      IREE_CUDA_DISABLE: 0
+      IREE_HIP_DISABLE: 1
+    runs-on:
+      - self-hosted # must come first
+      - runner-group=presubmit
+      - environment=prod
+      - a100
+      - os-family=Linux
+    steps:
+      - name: "Checking out repository"
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+      - name: "Checking out runtime submodules"
+        run: ./build_tools/scripts/git/update_runtime_submodules.sh
+      - name: Querying GPU information
+        run: |
+          ./build_tools/scripts/check_cuda.sh
+          ./build_tools/scripts/check_vulkan.sh
+      - name: "Checking disk usage"
+        run: df -h
+      - name: "Running a test command under docker"
+        run: |
+          ./build_tools/github_actions/docker_run.sh \
+              --env IREE_CPU_DISABLE \
+              --env IREE_VULKAN_DISABLE \
+              --env IREE_CUDA_DISABLE \
+              --env IREE_HIP_DISABLE \
+              gcr.io/iree-oss/nvidia@sha256:433e072f075f90fdf07471673626b17091d8d8e2395626f1e6ac6c98803c8807 \
+              echo "hi"
+      - name: "Checking disk usage"
+        run: df -h
+
+  test_nvidia_a100_postsubmit:
+    env:
+      IREE_CPU_DISABLE: 1
+      IREE_VULKAN_DISABLE: 0
+      IREE_CUDA_DISABLE: 0
+      IREE_HIP_DISABLE: 1
+    runs-on:
+      - self-hosted # must come first
+      - runner-group=postsubmit
```

Comment:
> Event type 'pull_request' is not allowed on this runner. Aborting workflow.
https://github.com/iree-org/iree/actions/runs/9520021624/job/26244438607?pr=17676#step:2:5

🤦 well, that make sense but makes debugging harder.

workflow_dispatch _is_ allowed, so I can try that
https://github.com/iree-org/iree/blob/c5d4b96a4aadbc86c6b63052e900adbc88037d38/build_tools/github_actions/runner/config/hooks/validate_trigger.postsubmit.sh#L13-L20